### DISCUSSION TITLE
Alpine Expression Error: Cannot read property 'cloneNode' of undefined

### DISCUSSION THREAD

**Discussion by giorgiopogliani** [alpinejs/alpine#1790] [3. Feature ideas]
I got this error because I was using x-if on a `div` and not on the `template`.  

Maybe, we could we catch this error and suggest the use of `template` tag?

**Comment by anjanesh** [comment]
Had the same issue and landed here - is this not documented in the official docs ?

**Reply by SimoTod** [reply]
@anjanesh second paragraph: https://alpinejs.dev/directives/if

**Reply by anjanesh** [reply]
Thank You.
